features/cli/run/symbolic_controlled_phase.feature.md

```
@@ -0,0 +1,91 @@
+  ```
+
+## Scenario: 未対応ゲートの制御版は明確なエラーを表示する
+
+- Given 次の circuit.json がある:
+
+  ```json
+  {
+    "cols": [["•", "Unknown"]],
+    "qubits": 2
+  }
+  ```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)